README.md

```diff
@@ -116,180 +113,170 @@ func main() {
 }
 ```
 
-More elaborate examples are available in the [examples](examples) directory,
-including [how to configure](examples/confluent_cloud_example) the Go client
-for use with [Confluent Cloud](https://www.confluent.io/confluent-cloud/).
-
+More elaborate examples are available in the [examples](examples) directory, including
+[how to configure](examples/confluent_cloud_example) the Go client for use with
+[Confluent Cloud](https://www.confluent.io/confluent-cloud/).
 
-Getting Started
-===============
+## Installation
 
-Installing librdkafka
----------------------
+### Installing librdkafka
 
-This client for Go depends on librdkafka v1.1.0 or later, so you either need to install librdkafka
```

Comment:
Don't remove/change this chapter, we shouldn't require people to visit some other page to get full instructions.